pkg/checks/external/util/main.go

```diff
@@ -88,4 +89,48 @@ func GetInstanceNamespace(defaultNamespace string) string {
 	}
 
 	return instanceNamespace
-}
\ No newline at end of file
+}
+
+// PodNameExists determines if a pod with the specified name exists in the specified namespace.
+func PodNameExists(client *kubernetes.Clientset, podName string, namespace string) (bool, error) {
```

Comment:
so we have another function `podExists` in this external/main.go file that almost does the same thing:
https://github.com/Comcast/kuberhealthy/blob/master/pkg/checks/external/main.go#L1189-L1216. Could we find where this function `podExists` is being used and replace with this one?